### PR TITLE
made m/f rule more generic

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: tests/test_general_cases/test_api_english_gender_acronyms/output.json [hunk 1]
```diff
@@ -8,16 +8,13 @@
...
                 {
-                    "text": "*/w/m"
+                    "text": "*/f/m"
                 },
                 {
-                    "text": "w/d/m"
-                },
-                {
-                    "text": "w/*/m"
+                    "text": "EEO"
```
Comment (lsmith77) [inline tests/test_general_cases/test_api_english_gender_acronyms/output.json]: @tje-ww accepting this alternative would then result in `Developer m/f` to become `Developer EEO` which imho doesn't make much sense. what was your idea here in regards to `EEO` ?

File: app/rules.py [hunk 2]
```diff
@@ -749,8 +710,7 @@
     "push the boundaries",
     "fossil promo",
     "good friends",
-    "good at"
-    "of a kind",
+    "good at" "of a kind",
```
Comment (lnazarenko) [inline app/rules.py]: missing comma  between "good at”  and "of a kind"

File: tests/test_general_cases/test_api_german_gender_acronyms_with_parenthesis/output.json [hunk 3]
```diff
@@ -8,20 +8,14 @@
                     "remove": true
                 },
                 {
-                    "text": "(d/w/m)"
+                    "text": "(*/M/W)"
                 },
```
Comment (lnazarenko) [inline tests/test_general_cases/test_api_german_gender_acronyms_with_parenthesis/output.json:12]: (*/M/W) should be W replaced with F

**Review by lnazarenko — APPROVED** [review]

**Comment by lsmith77** [comment]
@tje-ww the text also doesn't work well if `w` or `f` is the first letter

image

**Pull request merged**